Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -596,7 +596,14 @@ end
 				HasPluginChannel = { Params = "ChannelName", Return = "bool", Notes = "Returns true if the client has registered to receive messages on the specified plugin channel." },
 				IsUUIDOnline = { Params = "UUID", Return = "bool", IsStatic = true, Notes = "Returns true if the UUID is generated by online auth, false if it is an offline-generated UUID. We use Version-3 UUIDs for offline UUIDs, online UUIDs are Version-4, thus we can tell them apart. Accepts both 32-char and 36-char UUIDs (with and without dashes). If the string given is not a valid UUID, returns false."},
 				Kick = { Params = "Reason", Return = "", Notes = "Kicks the user with the specified reason" },
+        SendHideTitle = { Params = "", Return = "", Notes = "Sends the client to hide the title." },
 				SendPluginMessage = { Params = "Channel, Message", Return = "", Notes = "Sends the plugin message on the specified channel." },
+        SendResetTitle = { Params = "", Return = "", Notes = "Sends the client to reset the title." },
+        SendSetSubTitle = { Params = "SubTitle", Return = "", Notes = "Sends the client the subtitle." },
+        SendSetRawSubTitle = { Params = "RawSubTitle", Return = "", Notes = "Sends the client the raw subtitle." },
+        SendSetTitle = { Params = "Title", Return = "", Notes = "Sends the client the title." },
+        SendSetRawTitle = { Params = "RawTitle", Return = "", Notes = "Sends the client the raw title." },
+        SendTitleTimes = { Params = "FadeInTicks, DisplayTicks, FadeOutTicks", Return = "", Notes = "Sends the client the title times and shows the title." },
```

Comment:
The wording is unfortunate.
`Send X to Y` is better.
Still, if I remember correctly, the title functions are a little bit unintuitive in that they don't cause the title to be displayed immediately, perhaps that should be documented as well and a better phrasing should be used to indicate that.